projects/go-lib/src/styles/_forms.scss

```diff
@@ -128,6 +128,10 @@ $input--disabled-background: rgba($base-dark-secondary, .4);
   background-color: transparent;
   color: $theme-dark-color;
 
+  option {
```

Comment:
@eligolding your second point is intended. This make is easy for us to determine where the styles come from. If we just have `option` styled, that's where we run into issues with the cascade and specificity. This is the exact reason why we want to use BEM.

Reply:
I don't disagree with the rule in general, I'm just saying that we need an elegant solution to this bug. This is a bug in the way _we_ implemented the styles applied to the select when using the `.go-form__select--dark` class. We are breaking the options from rendering properly, which I don't think should then require the user to fix.

>This make is easy for us to determine where the styles come from... If we just have option styled, that's where we run into issues with the cascade and specificity. 

This gets converted to a descendant selector, it will only be applied to options of a select using `.go-form__select--dark`.

<img width="333" alt="Tangoe 2019-09-16 12-26-43" src="https://user-images.githubusercontent.com/5261826/64975587-621ba000-d87d-11e9-8d3c-d579c64d68f3.png">

Reply:
@eligolding I see what you're getting at now. This is not the select component, but we cannot apply those classes with a change of binding automatically for the implementer. I'm okay if you'd like to proceed with this for now, but because we've rolled out the `go-select` component in #142 we probably won't need this once the `go-select` is adopted. We can always revisit then.